docs/source/user_guide.rst

```diff
@@ -208,9 +202,23 @@ EinsteinPy also supports smbolic calculations in :py:class:`~einsteinpy.utils.ch
         [0, 0, -r*(-a/r + 1), 0]
 
 
+    .. code-block:: python
+
+        import sympy
+        from einsteinpy.symbolic import SchwarzschildMetric, EinsteinTensor
+
+        m = SchwarzschildMetric()
+        G1 = EinsteinTensor.from_metric(m)
+        print(G1.arr)
+
+    .. code-block:: python
+
+        [[a*c**2*(-a + r)/r**4 + a*c**2*(a - r)/r**4, 0, 0, 0], [0, a/(r**2*(a - r)) + a/(r**2*(-a + r)), 0, 0], [0, 0, 0, 0], [0, 0, 0, 0]]
```

Comment:
Have you calculated it? Is it the actual result?

Reply:
Yes :)
![image](https://user-images.githubusercontent.com/51050291/64074919-df59e900-ccb1-11e9-88a4-703258424cd9.png)